stdlib/public/core/Set.swift

```diff
@@ -415,7 +415,7 @@ extension Set: Equatable {
     case (true, false):
       return lhs._variant.asNative.isEqual(to: rhs._variant.asCocoa)
     case (false, true):
-      return rhs._variant.asNative.isEqual(to: lhs._variant.asCocoa)
```

Comment:
I was knee deep in Set guts trying to figure out how this managed to remain undetected for so long, but it turns out the original code is correct! Unlike the rest of the cases, here `isEqual(to:)` is called on `rhs`, not `lhs`. (`_CocoaSet.isEqual(to:)` does not support a native argument, so this line wouldn't compile with the suggested change.)

This is way too subtle. I blame the terrible naming convention `lhs` and `rhs` -- naming these parameters `left` and `right` would've highlighted the swap much better. Would you like to rename them?

Reply:
oh! I see, in this case the `lhs` and `rhs`were reversed! The naming does make it hard to notice that. Will rename, sorry for the false alarm